### PR TITLE
Explain how to disable GitHub Actions CI Viz integration

### PR DESCRIPTION
**Pull request by cg14823**
### What does this PR do?

Add instructions on how to disable the GitHub Actions CI Visibility integration

### Motivation

The process is not obvious so customers have been having doubts on how to do this.

### Preview

https://docs-staging.datadoghq.com/carlos.gonzalez/add-gha-disable-instructions/continuous_integration/setup_pipelines/github/#disabling-github-actions-tracing

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by juan-fernandez — APPROVED** [review]

File: content/en/continuous_integration/setup_pipelines/github.md
Comment (jtappa) [inline content/en/continuous_integration/setup_pipelines/github.md:48]:
```suggestion
To disable the CI Visibility GitHub Actions integration, make sure the GitHub app is no longer subscribed to the
```

File: content/en/continuous_integration/setup_pipelines/github.md
Comment (jtappa) [inline content/en/continuous_integration/setup_pipelines/github.md:53]: I'm trying to follow along with a quick tester github app. Is this what you're referring to?
Screen Shot 2022-03-29 at 10 11 39 AM

if so, we should match their UI text exactly to be this:
```suggestion
3. Scroll to the **Subscribe to events** section, and make sure that **Workflow job** and **Workflow run** are not selected.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Comment by jtappa** [comment]
Is this okay to merge, or did you want to wait for a review from @DataDog/ci-app-backend ?

**Comment by cg14823** [comment]
> Is this okay to merge, or did you want to wait for a review from https://github.com/orgs/DataDog/teams/ci-app-backend ?

It is ok to go through with out it.

**Pull request merged**
